### ISSUE TITLE
data.py

### ISSUE THREAD

**Issue by RichardShea** [BenjaminSischka/SBSGMest#1]
Hi, any links the data (files) referenced in data.py?

Thank you,

-Richard

**Comment by BenjaminSischka** [comment]
Hi Richard,

yeah, sure. You can find the links to the files for the political blog, military alliance, and human brain functional coactivation network in the the readme file.

Hope that's what you meant.

Best, Ben

**Comment by RichardShea** [comment]
Hi Ben,

I was able to find and bring in the human brain (_Coactivation_matrix.mat_) into the _application.py_ file (setting simulate = False and set data_ = 'brain').  

However, I'm having trouble finding/bringing in the military alliance data.  Did you preprocess the data from [atopdata](http://www.atopdata.org/data.html)?  

I'm can't find the _country_list.csv, alliances_strong_post_g.csv, alliances_strong_post_el.csv, and alliances_strong_post_adjMat_2016.csv_ files referenced in `def GraphFromData` (data.py).

Thank you,

Richard

**Comment by BenjaminSischka** [comment]
Yes, you're totally right, sorry. I did preprocess the alliance data set. 

To get a meaningful network, I've picked only 'strong' alliances that where in force in 2016 as the most recent available year.
